kernel/src/process_load.rs

```diff
@@ -0,0 +1,798 @@
+//! Reading process binaries and loading them into in-memory Tock processes.
```

Comment:
Maybe `process_loading.rs`?

Reply:
Yes, makes sense, to sync it with checking.